<launch>
 <!-- Nodelet manager for this pipeline -->
  <node name="update_frame_id" pkg="depth_image_proc" type="update_frame_id.py" output="screen" />
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
  <node pkg="nodelet" type="nodelet" name="nodelet1" args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <!-- Input: Camera calibration and metadata. --> 
    <remap from="camera_info" to="/camera/depth/camera_info"/>
    
    <!-- Input: Rectified depth image, registered to the Depth camera. -->
    <remap from="image_rect" to="/DepthBB"/>
    
    <!-- Output: XYZ point cloud. -->
    <remap from="points" to="/camera/depth/points"/>
  </node>
</launch>